lumibot/backtesting/polygon_backtesting.py

```diff
@@ -202,11 +204,7 @@ def _pull_source_symbol_bars(
         start_dt, ts_unit = self.get_start_datetime_and_ts_unit(length, timestep, current_dt, start_buffer=START_BUFFER)
 
         # Get data from Polygon
-        pandas_data_update = self.update_pandas_data(asset, quote, length, timestep, start_dt)
-
-        if pandas_data_update is not None:
-            # Add the keys to the self.pandas_data dictionary
-            self.pandas_data.update(pandas_data_update)
+        self.update_pandas_data(asset, quote, length, timestep, start_dt)
```

Comment:
why is update_data_store=False here? shouldnt we also update the data store here if needed?

Reply:
As above.  Would be happy to here from or chat with someone who knows how this is supposed to work.